docker/Dockerfile

```diff
@@ -10,7 +10,9 @@ RUN apk update && \
         mkdir -p /var/jenkins_home/init.groovy.d && \
         git clone https://github.com/awslabs/git-secrets.git && \
         make install -C git-secrets && \
-        git-secrets --register-aws --global
+        git-secrets --register-aws --global && \
+        git-secrets --add --global '([^0-9])*[0-9]{12}([^0-9])*' && \
+        git-secrets --add --global -a '([^0-9])*123456789012([^0-9])*'
```

Comment:
Nice - I'm glad we can do this globally rather than having to update every call to git-secrets. 👍 

These values aren't very obvious - I think it's worth a comment explaining that they're looking for AWS account numbers, and that we're excluding "123456789012" because it's used as a dummy account number in documentation.